docs/Use-Case-Cyclic-Voltammetry.md

```diff
@@ -0,0 +1,20 @@
+Use Case – cyclic voltammetry experiment for three electrode cell
+
+1. Input Initial Potential (V vs. Eocv)
+2. Input first turnover potential (V vs. Ref)
```

Comment:
If you make these all 1. then the markdown will number them correctly when it renders.  It is nice because you can reorder everything without retyping numbers.  Give it a try.  It will also render correctly the way you have typed it!